myst_nb/core/read.py

```diff
@@ -68,7 +68,7 @@ def create_nb_reader(
     # we check suffixes ordered by longest first, to ensure we get the "closest" match
     iterator = sorted(readers.items(), key=lambda x: len(x[0]), reverse=True)
     for suffix, (reader, reader_kwargs, commonmark_only) in iterator:
-        if path.endswith(suffix):
+        if Path(path).suffix == suffix:
```

Comment:
Thanks for this. I guess `Path(path)` is for compatibility with older versions of `sphinx`, correct? Because for later versions `path` is already a `Path` object. Is this change available on `conda`? If not then by when can we expect this to be there?

Reply:
cc: @agoose77

Reply:
Yes, the commit message says clearly that this was needed for compatibility.

Reply:
Awesome. Thankd a lot for this. 😃